docs/community/maintainer/pr-review.md

```diff
@@ -0,0 +1,16 @@
+# Pull Request Review Policy
+
+This document outlines the review policy for pull requests in the Trivy project.
+
+## Core Principles
+
+### 1. All Changes Through Pull Requests
+All changes to the `main` branch must be made through pull requests. Direct commits to `main` are not allowed.
+
+### 2. Required Approvals
+Every pull request requires approval from at least one CODEOWNER before merging.
```

Comment:
what about case when PR related with different CODEOWNER's?
e.g. #8959
This PR changes:
- `pkg/misconfig` (@simar7 @nikpivkin)
- `pkg/fanal` (@knqyf263 @DmitriyLewen)

PR owner should wait 2 approvals, (for each section), right?

Reply:
Added https://github.com/aquasecurity/trivy/pull/9032/commits/bc48cef99fe320d2a47d892d276479f0e11f4a31